### PR TITLE
part1_sedf

### PR DESCRIPTION
**Pull request by mohi9282**
New guide for SEDF: part1_sedf. Resolves issue /ArcGIS/geosaurus/issues/6847

### CODE REVIEW

**Comment by mohi9282** [comment]
@achapkowski the sedf guide part-1 is ready for your review. Atma has reviewed this already. The goal is to publish it with 1.9.1 release.

cc: @AtmaMani

**Review by achapkowski — COMMENTED** [review]
I normally refer to Spatially enabled DataFrame as **SeDF** not SEDF

**Review by achapkowski — APPROVED** [review]

File: guide/05-working-with-the-spatially-enabled-dataframe/part1_introduction_to_sedf.ipynb [hunk 1]
```diff
@@ -0,0 +1,1555 @@
+{
```
Comment (AtmaMani) [inline guide/05-working-with-the-spatially-enabled-dataframe/part1_introduction_to_sedf.ipynb:1]: <p>Add a Note here using the same cell style as in the note at the very bottom:</p><p> </p><p>"</p><p>Note: Spatial Data Engineering using SeDF builds on top of core Data Engineering concepts in Python. If you are new to Pandas, NumPy and related libraries, we recommend you start with the <a href="part1-introduction-to-dataengineering/" target="_blank">Introduction to Data Engineering</a> guide series and then come here.</p><p>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (AtmaMani) [inline guide/05-working-with-the-spatially-enabled-dataframe/part1_introduction_to_sedf.ipynb:1]: <p><em>"GeoSeriesAccessor</em> and <em>GeoAccessor</em> classes are similar to <a href="https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.html#pandas.Series" target="_blank">pandas.Series</a> and <a href="https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html#pandas.DataFrame" target="_blank">pandas.DataFrame</a> objects however they do not need to be called repeatedly during the analysis for usage.  </p><p>Importing Geo will add spatial and geom namespaces to your dataframe and series resp. It is best practice to import Pandas followed by importing both the <em>GeoAccessor</em> and the <em>GeoSeriesAccessor</em> classes at the beginning of the analysis. Importing these classes registers their functionality with Pandas and allows you to start performing spatial operations. You may import the classes as follows: "  ---&gt; </p><p> </p><p>"<em>GeoSeriesAccessor</em> and <em>GeoAccessor</em> classes are similar to <a href="https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.html#pandas.Series" target="_blank">pandas.Series</a> and <a href="https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html#pandas.DataFrame" target="_blank">pandas.DataFrame</a> objects. However, you do not work with them directly. Instead, you import them right after you import Pandas as shown in the snippet below. Importing these classes registers the spatial functionality with Pandas and allows you to start performing spatial operations on your DataFrames.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (AtmaMani) [inline guide/05-working-with-the-spatially-enabled-dataframe/part1_introduction_to_sedf.ipynb:1]: <p>"However, we recommend using <code>arcpy</code> for better accuracy <strong>and support for a wider gamut of data sources"</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (AtmaMani) [inline guide/05-working-with-the-spatially-enabled-dataframe/part1_introduction_to_sedf.ipynb:1]: <p>The <code>area</code> property retrives the area of each buffer <strong>in the units of the DataFrame's spatial reference.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Comment (AtmaMani) [inline guide/05-working-with-the-spatially-enabled-dataframe/part1_introduction_to_sedf.ipynb:1]: <p>Let's <strong>now </strong>perform some basic Pandas operations on a SeDF. <strong>One of the benefits of the accessor pattern in SeDF is, the SeDF object is still of type 'DataFrame'. Thus, you can continue to perform regular Pandas DataFrame operations.</strong></p><p> </p><p>We will:</p><p>....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Comment (AtmaMani) [inline guide/05-working-with-the-spatially-enabled-dataframe/part1_introduction_to_sedf.ipynb:1]: <p>Insert a plug to the primer guide series here. "If you would like to learn more about Pandas and data engineering with Pandas, checkout our [Data Engineering primer guide](part3-introduction-to-pandas/)</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

**Review by AtmaMani — CHANGES_REQUESTED** [review]
@mohi9282 this is an excellent guide! I have asked for minor revisions, mostly text.

**Comment by mohi9282** [comment]
@AtmaMani I have updated the sample per the requested changes.

**Review by AtmaMani — APPROVED** [review]

**Pull request merged**